components/vocabularies/hpo/api/src/main/java/org/phenotips/vocabulary/internal/solr/HumanPhenotypeOntology.java

```diff
@@ -86,6 +102,12 @@ public String getName()
     }
 
     @Override
+    public Collection<String> getSupportedCategories()
+    {
+        return Collections.unmodifiableCollection(SUPPORTED_CATEGORIES);
```

Comment:
Since the categories never change, it would be better to make the value of `SUPPORTED_CATEGORIES` unmodifiable from the start.